pandas/tests/frame/test_validate.py

```diff
@@ -39,3 +39,26 @@ def test_validate_bool_args(self, dataframe, func, inplace):
 
         with pytest.raises(ValueError, match=msg):
             getattr(dataframe, func)(**kwargs)
+
+    @pytest.mark.parametrize(
+        "func",
+        [
+            "reset_index",
+        ],
+    )
+    @pytest.mark.parametrize("inplace", [1, "True", [1, 2, 3], 5.0])
```

Comment:
Can you also test for false?

Reply:
This one doesn't check for `True` either, but for `"True"` as a string

Reply:
Missed that, sorry